tests/unit_tests/sparse_blas/source/sparse_gemm_buffer.cpp

```diff
@@ -214,41 +233,67 @@ void test_helper(sycl::device *dev, oneapi::mkl::transpose transpose_A,
  * 
  * @tparam fpType Complex or scalar, single or double precision type
  * @param dev Device to test
+ * @param num_passed Increase the number of configurations passed
+ * @param num_skipped Increase the number of configurations skipped
  */
 template <typename fpType>
-void test_helper_transpose(sycl::device *dev) {
+void test_helper_transpose(sycl::device *dev, int &num_passed, int &num_skipped) {
     std::vector<oneapi::mkl::transpose> transpose_vals{ oneapi::mkl::transpose::nontrans,
                                                         oneapi::mkl::transpose::trans };
     if (complex_info<fpType>::is_complex) {
         transpose_vals.push_back(oneapi::mkl::transpose::conjtrans);
     }
     for (auto transpose_A : transpose_vals) {
         for (auto transpose_B : transpose_vals) {
-            test_helper<fpType>(dev, transpose_A, transpose_B);
+            test_helper<fpType>(dev, transpose_A, transpose_B, num_passed, num_skipped);
         }
     }
 }
 
 TEST_P(SparseGemmBufferTests, RealSinglePrecision) {
     using fpType = float;
-    test_helper_transpose<fpType>(GetParam());
+    int num_passed = 0, num_skipped = 0;
+    test_helper_transpose<fpType>(GetParam(), num_passed, num_skipped);
+    if (num_skipped > 0) {
+        // Mark that some tests were skipped
+        GTEST_SKIP() << "Passed: " << num_passed << ", Skipped: " << num_skipped
+                     << " configurations." << std::endl;
```

Comment:
```suggestion
        // Mark that some tests were skipped
        // This may not print/work well with ctest. However, running the binary
        // files of the tests directly with the `--terse-output` flag prints this
        // properly. E.g., ./bin/test_main_sparse_blas_ct --terse-output
        GTEST_SKIP() << "Passed: " << num_passed << ", Skipped: " << num_skipped
                     << " configurations." << std::endl;
```

Reply:
I'm also fine with it being added in the first `GTEST_SKIP()` occurrence in each individual file (which would be 6 files right now, I believe).